src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import bookReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: bookReducer,
+  categories: categoriesReducer,
+});
+
+export default rootReducer;
```

Comment:
- [ ] Great job combining reducers according to Redux we are just missing to createStore to be able to use it later in our view, here's what we are missing, let's kindly implement it 
```js
const store = createStore("combinedReducers")
```
you can read more about it [here](https://redux.js.org/api/createstore)

Reply:
Hi @Amrhub , 

Thank you for your comments. 

- But the part of your comment is included in the index.js file in the `src/` folder. 
- This submission is working as intended, you may check this by cloning the project locally.

Reply:
HI @mengiefen 

Oh I see, I would suggest you move it to `configureStore.js` so it would be the place to have all settings in there 😇